modin/pandas/test/dataframe/test_iter.py

```diff
@@ -284,7 +295,7 @@ def test_constructor(data):
     [
         np.arange(1, 10000, dtype=np.float32),
         [
-            pd.Series([1, 2, 3], dtype="int32"),
+            pandas.Series([1, 2, 3], dtype="int32"),
             pandas.Series([4, 5, 6], dtype="int64"),
```

Comment:
why changed? Isn't the initial idea of this test was to check that the object can be created from both: pandas and modin Series?

Reply:
I explained this in your comment on test_groupby. I changed this back to a modin series.